pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -170,6 +170,11 @@ func (c *Cluster) initializeSecureDashboard() (bool, error) {
 			return false, nil
 		}
 	}
+	if c.dashboard.ReadOnly {
+		if err := c.setAdminReadOnly(); err != nil {
+			return false, errors.Wrapf(err, "failed to set login role for the ceph dashboard")
```

Comment:
```suggestion
			return false, errors.Wrap(err, "failed to set login role for the ceph dashboard")
```

Reply:
Is the error message accurate? Is this really what this method does? I'd expect a mention of "read-only" here.